website/docs/concepts/components.md

```diff
@@ -213,6 +213,58 @@ After Components are unmounted from the DOM, Yew calls the `destroy` lifecycle m
 necessary if you need to undertake operations to clean up after earlier actions of a component
 before it is destroyed. This method is optional and does nothing by default.
 
+### Infinite loops
+
+Infinite loops are possible with yew's lifecycle methods, but are only caused when trying to update
```

Comment:
Not sure how consistent we are with this, but I think in most cases Yew is capitalized.

```suggestion
Infinite loops are possible with Yew's lifecycle methods, but are only caused when trying to update
```